migrations/7_fungible_balance.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE fungible_tokens ADD COLUMN fbt_balance bigint NOT NULL DEFAULT 0;
+ALTER TABLE fungible_tokens ADD COLUMN fbt_slot_updated bigint NOT NULL DEFAULT 0;
+CREATE INDEX fungible_tokens_fbt_balance_idx ON fungible_tokens(fbt_balance) WHERE fbt_balance > 0;
```

Comment:
do we delete and recreate this index in the full sync? The idea to drop all the indexes is to ensure PG is not updating any indexes while loading from csv and only creates it once, when all the data is in

Reply:
Yeah, i forget to add new indexes into drop_indexes() function. Thank you!